tests/testqPrefCloudStorage.cpp

```diff
@@ -80,11 +84,13 @@ void TestQPrefCloudStorage::test_set_load_struct()
 	tst->set_cloud_storage_email("t3 email");
 	tst->set_cloud_storage_email_encoded("t3 email2");
 	tst->set_save_password_local(true);
+        tst->set_cloud_auto_sync(true);
```

Comment:
whitespace issues - either in your code or the existing code... here, above, and below

Reply:
yep, resolved the whitespace issues.